src/consts/config.ts

```diff
@@ -11,7 +12,8 @@ interface Config {
   showTipBox: boolean; // Show/Hide the blue tip box above the transfer form
   showDisabledTokens: boolean; // Show/Hide invalid token options in the selection modal
   walletConnectProjectId: string; // Project ID provided by walletconnect
-  withdrawalWhitelist: string; // comma-separated list of CAIP2 domains to which transfers are supported
+  withdrawalWhitelist: string; // comma-separated list of CAIP2 chain IDs to which transfers are supported
```

Comment:
NIt: Can we just make them similar? They feel just different enough

Reply:
idk I think both serve their purpose well enough as they are, i'm gonna merge so we can ship the change but am not opposed to changing